File: foundation 3 month/lesson3/homework/ex_4.py
a=input("Harfni kiriting: ")
k=ord(a)
if(k>=97 and k<=122):
	d=k-32
	print(chr(d))
elif(k>=65 and k<=90):
	print(a)
else:
	print("Bunday harf yuq") 

File: full stack/full stack 1 month/lesson6/homework/ex_4.js
let harf = "Z"
if (harf.charCodeAt(0)>=65 && harf.charCodeAt(0)<=90 && harf.length === 1)
{
    let s =harf.charCodeAt(0)+32
    console.log(String.fromCharCode(s))
}
else
{
    console.log("Harf kiriting")
}